### PR TITLE
Handle idle timeout failures in az aks browse

### LINKED ISSUE #11580: az aks browse should handle idle timeouts
Opening a clean bug to track the fact that `az aks browse` does not currently handle the default idle timeout from the Azure load balancer (4 minutes). Reports of this problem are currently intermingled with other related issues.

Expected behavior: browse command handles idle timeouts and automatically re-establishes the proxy

Actual behavior: connection drops and dashboard is no longer accessible in the browser

### PR DESCRIPTION
**Pull request by seanmck**
This change addresses periodic connection failures in `az aks browse` due to Azure LB timeouts (every 4-5 minutes). It does so by introducing a reconnect loop that is only broken when the user explicitly exits via ctrl-c.

Closes #11580 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@qwordy please take a look.

**Comment by qwordy** [comment]
You have style error:
/home/vsts/work/1/s/src/azure-cli/azure/cli/command_modules/acs/custom.py:1572:41: E128 continuation line under-indented for visual indent
4     E128 continuation line under-indented for visual indent
4     W293 blank line contains whitespace

**Comment by seanmck** [comment]
@qwordy I believe I've resolved all style issues. Can you take another look?

**Comment by qwordy** [comment]
ERROR: ************* Module azure.cli.command_modules.acs.custom
src/azure-cli/azure/cli/command_modules/acs/custom.py:1498:0: R0912: Too many branches (23/20) (too-many-branches)
src/azure-cli/azure/cli/command_modules/acs/custom.py:1498:0: R0912: Too many branches (23/20) (too-many-branches)

**Comment by seanmck** [comment]
Fixes #11580

**Comment by MyronFanQiu** [comment]
@seanmck Hi. Any update?

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -387,6 +387,10 @@ Release History
 
 * Enhanced error detail for vault delete in force mode.
 
+**AKS**
```
Comment (yungezz) [inline src/azure-cli/HISTORY.rst]: history.rst is automatically generated now, pls add fix description in PR title. thanks.

**Comment by yungezz** [comment]
hi @seanmck  are you still on this PR? could you pls resolve conflict and comments? thanks.

**Pull request closed without merging**